web/src/components/search/SearchResult.vue

```diff
@@ -4,14 +4,16 @@
       <q-table
         ref="searchTable"
         v-model:expanded="searchResult._source"
+        v-model:pagination="pagination"
         data-cy="search-result-area"
         :rows="searchResult"
         :columns="resultColumns"
         :loading="searchLoading"
-        :pagination="pagination"
+        :rows-per-page-options="[5, 10, 20, 50, 100, 500, 1000, 5000, 10000]"
```

Comment:
I think page options just have `[5, 10, 20, 50, 100]` is ok, it looks no need very large page. And you should use a variable to set it.